services/brig/deb/opt/brig/templates/en/team/email/invitation.txt

```diff
@@ -3,16 +3,21 @@
 ${brand_label_url} [${brand_url}]
 
 TEAM INVITATION
-${inviter} has invited you to join a team on ${brand}. Click the button below to
-accept the invitation.
+${inviter} has invited you to join the team ${team_name} on ${brand}. Click the
```

Comment:
We removed `team_name` from the messages at some point in the past: it's prone to abuse, as the team name can be anything, including a link to a dubious website or offensive language.

See also https://github.com/wireapp/wire-server/pull/136 and the item `Removal of user generated text input from team invitations`. It's a security/phishing concern.